lib/errors/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  NotFoundError: require('./not-found'),
+  BadRequestError: require('./bad-request')
+};
```

Comment:
Use the errors directly from `@asl/service/errors`

Reply:
ooh, shiny